cmd/mesh/manifest-apply.go

```diff
@@ -65,8 +66,8 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
 		Args:  cobra.ExactArgs(0),
 		Run: func(cmd *cobra.Command, args []string) {
 			l := newLogger(rootArgs.logToStdErr, cmd.OutOrStdout(), cmd.OutOrStderr())
-			if !maArgs.yes && maArgs.kubeConfigPath == "" && maArgs.context == "" {
-				if !confirm("Are you sure?", cmd.OutOrStdout()) {
+			if !maArgs.skipConfirmation && maArgs.kubeConfigPath == "" && maArgs.context == "" {
+				if !confirm("Do you wish to run this command? (Y/N)", cmd.OutOrStdout()) {
```

Comment:
Sorry, this is real nitpicking but I suggest "Proceed?" instead.
"Do you wish to run this command" suggests in some sense that the command might be optional.